packages/@uppy/tus/src/index.js

```diff
@@ -256,7 +256,7 @@ module.exports = class Tus extends Plugin {
         }
       }
 
-      /** @type {{ [name: string]: string }} */
+      /** @type {Record<string, string>} */
```

Comment:
Working around a possible limitation in the JSDoc plugin's typescript syntax checking